Removed the layout item referencing Account.SLAExpirationDate__c; all unrelated metadata and ordering were preserved.

--- before/backup/layouts/Account-Account %28Support%29 Layout.layout-meta.xml
+++ after/working/layouts/Account-Account %28Support%29 Layout.layout-meta.xml
@@ -52,10 +52,6 @@
             <layoutItems>
                 <behavior>Edit</behavior>
                 <field>SLA__c</field>
-            </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>SLAExpirationDate__c</field>
             </layoutItems>
             <layoutItems>
                 <behavior>Edit</behavior>
@@ -426,4 +422,4 @@
         <sizeY>0</sizeY>
         <summaryLayoutStyle>Default</summaryLayoutStyle>
     </summaryLayout>
-</Layout>
+</Layout>
\ No newline at end of file
